src/metrics/nom.rs

```diff
@@ -123,6 +123,7 @@ impl Stats {
     /// Returns the average number of function definitions over all spaces
     #[inline(always)]
     pub fn functions_average(&self) -> f64 {
+        println!("SPACE {}", self.space_count);
```

Comment:
Please remove the debugging statement ;)

Reply:
Bah! Forgot about it. Done.